make install-apis sometimes fails

`make install-apis` intermittently fails with the following error:

```
kubectl apply -f deploy/crds/
customresourcedefinition.apiextensions.k8s.io/buildruns.shipwright.io created
customresourcedefinition.apiextensions.k8s.io/builds.shipwright.io created
customresourcedefinition.apiextensions.k8s.io/buildstrategies.shipwright.io created
customresourcedefinition.apiextensions.k8s.io/clusterbuildstrategies.shipwright.io created
# Wait for the CRD type to be established; this can take a second or two.
kubectl wait --timeout=10s --for condition=established crd/clusterbuildstrategies.shipwright.io
error: .status.conditions accessor error: <nil> is of the type <nil>, expected []interface{}
make: *** [Makefile:231: install-apis] Error 1
Error: Process completed with exit code 2.
```

We should address this as it causes PR runs to fail. In particular this `kubectl wait` fails:

https://github.com/shipwright-io/build/blob/5afa4c37c56f6c0c34e010327dae1550e53a9749/Makefile#L231

--- a/Makefile
+++ b/Makefile
@@ -66,6 +66,9 @@ TEST_PRIVATE_GITLAB ?=
 # private repository authentication secret
 TEST_SOURCE_SECRET ?=
 
+# default timeout for kubectl and other interactions with external systems
+TIMEOUT ?= 30s
+
 # Image settings for building and pushing images
 IMAGE_HOST ?= ghcr.io
 IMAGE_NAMESPACE ?= shipwright-io/build
@@ -221,8 +224,11 @@ install-with-pprof:
 
 install-apis:
 	kubectl apply -f deploy/crds/
-	# Wait for the CRD type to be established; this can take a second or two.
-	kubectl wait --timeout=10s --for condition=established crd/clusterbuildstrategies.shipwright.io
+	for i in 1 2 3 ; do \
+		kubectl wait --timeout=$(TIMEOUT) --for="condition=something" crd/clusterbuildstrategies.shipwright.io && \
+		break ; \
+		sleep 15 ; \
+	done
 
 install-controller: install-apis
 	@echo "Building Shipwright Build controller for platform ${GO_OS}/${GO_ARCH}"

